orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltErrorResolver.java

```diff
@@ -49,12 +55,15 @@ public void init() {
             String file = FileReaderUtils.readFileFromClasspath("salt/errormessages.yaml");
             errorMessages = new Yaml().load(file);
             LOGGER.info("Error messages for salt: {}", errorMessages);
+            file = FileReaderUtils.readFileFromClasspath("salt/stderrcommands.yaml");
+            commandsWithStderrFailures = new Yaml().load(file);
+            LOGGER.info("Salt commands that will pull the failure from stderr: {}", commandsWithStderrFailures);
```

Comment:
Do we know when the error is available in salt/errormessages.yaml and salt/stderrcommands.yaml files?

Reply:
I'm not sure I understand the question. In errormessages.yaml, the code is basically replacing any errors run by the listed processes with the error in the YAML file. In stderrcommands.yaml we're just listing some processes, and the error comes from the stderr output. If the stderr output doesn't exist, then it defaults to a message that just says, "Failed to execute: <process string>"